automation_tools/__init__.py

```diff
@@ -2860,13 +2860,16 @@ def foreman_debug(tarball_name=None, local_path=None):
 # =============================================================================
 # Helper functions
 # =============================================================================
-def katello_installer(debug=False, distribution=None, verbose=True,
+def katello_installer(distribution=None, verbose=True,
                       sat_version='6.3', scenario=None, **kwargs):
     """Runs the installer with ``kwargs`` as command options."""
     # capsule-dns-forwarders should be repeated if setting more than one
     # value check if a list is being received and repeat the option with
     # different values
-    extra_options = []
+    extra_options = [
+        '--log-level=DEBUG',
+        '--foreman-proxy-log-level=DEBUG',
+    ]
```

Comment:
Please add debug for foreman `--foreman-logging-level=debug`
here debug has to be lowercase !!!